types/microsoft-graph/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove all the rules here since we expect all new package to fix the errors and adhere to the lint rules of the DT

Reply:
Okay, will fix those issues.

Reply:
@sheetalkamat Removed overwritten lint rules and fixed those errors.